packages/core/src/Stores/client-store.js

```diff
@@ -60,7 +60,7 @@ export default class ClientStore extends BaseStore {
 
     @observable local_currency_config = {
         currency: '',
-        decimal_places: '',
+        decimal_places: undefined,
```

Comment:
is the type for decimal places always changes?

Reply:
what do you mean? it's type number, so we can't assign empty string, it should be undefined if it doesn't have a value yet

Reply:
why not 0?

Reply:
usually in the code we are using empty string for strings or undefined for numbers, we don't use any default value if we are going to wait for it from the API

Reply:
i see, okay 👍